src/transformers/utils/hub.py

```diff
@@ -97,16 +110,26 @@ def is_offline_mode():
         " '~/.cache/huggingface/transformers' to avoid redownloading models you have already in the cache. You should"
         " only see this message once."
     )
-    shutil.move(old_default_cache_path, default_cache_path)
+    shutil.move(old_default_cache_path, constants.HF_HUB_CACHE)
 
-PYTORCH_PRETRAINED_BERT_CACHE = os.getenv("PYTORCH_PRETRAINED_BERT_CACHE", default_cache_path)
-PYTORCH_TRANSFORMERS_CACHE = os.getenv("PYTORCH_TRANSFORMERS_CACHE", PYTORCH_PRETRAINED_BERT_CACHE)
-HUGGINGFACE_HUB_CACHE = os.getenv("HUGGINGFACE_HUB_CACHE", PYTORCH_TRANSFORMERS_CACHE)
```

Comment:
Won't this break things if users have `HUGGINFACE_HUB_CACHE` set in their env?

Reply:
Nvm - I saw the discussion here: https://github.com/huggingface/transformers/pull/27564#issuecomment-1819097559